custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -633,8 +633,8 @@ def test_system_usage_export(self):
             SystemUsageExport(
                 config={
                     'domain': 'icds-cas',
-                    'month': datetime(2017, 5, 1)
-                }
+                    'month': datetime(2017, 5, 1),
```

Comment:
Can you add separate test case for beta=True instead of updating existing ones, So that we can ensure that existing report is not changed with the code change.

Reply:
okay